src/audio/sr_audio/sr_audio.c

```diff
@@ -0,0 +1,720 @@
+// Copyright(c) 2021 Sound Research Corporation. All rights reserved.
+#include <sof/audio/component.h>
+#include <sof/audio/buffer.h>
+#include <sof/audio/sr_audio/sr_audio.h>
+#include <sof/audio/format.h>
+#include <sof/audio/pipeline.h>
+#include <sof/common.h>
+#include <sof/debug/panic.h>
+#include <sof/ipc/msg.h>
+#include <sof/lib/alloc.h>
+#include <sof/lib/memory.h>
+#include <sof/lib/uuid.h>
+#include <sof/list.h>
+#include <sof/platform.h>
+#include <sof/string.h>
+#include <sof/ut.h>
+#include <sof/trace/trace.h>
+#include <ipc/control.h>
+#include <ipc/stream.h>
+#include <ipc/topology.h>
+#include <user/trace.h>
+#include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
+
+//#define _ADD_TEST_CLICK_AT48
+
+static const struct comp_driver comp_sraudio;
+
+// {9C5B18CC-12A5-4904-A84A-7E1FE86E0927}
+DECLARE_SOF_RT_UUID("sraudio", sraudio_uuid, 0x9c5b18cc, 0x12a5, 0x4904, 0xa8, 0x4a, 0x7e, 0x1f, 0xe8, 0x6e, 0x9, 0x27);
+
+DECLARE_TR_CTX(sraudio_tr, SOF_UUID(sraudio_uuid), LOG_LEVEL_INFO);
+
+#define PARAM_CONTEXT_SIZE   55000 // iSST
```

Comment:
best to prefix generic sounding macros so they wont collide e.g. I would use ```SR_``` prefix your macros

Reply:
Thought prefixing macros in C file wasn't necessary since the chance of collision is pretty low, will do as you say though.